sql-select/cheap-to-rent.sql

```diff
@@ -0,0 +1 @@
+SELECT "filmId", "title" from "films" where "rentalRate" > '1' LIMIT 50;
```

Comment:
I'm not sure this works. `rentalRate` is an integer but you are comparing it to a string. Also, you are looking for items with a rate greater than 1.